Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.13 AS build-env
+WORKDIR /go/src/github.com/fairwindsops/pluto/
+
+ENV GO111MODULE=on
+ENV GOPROXY=https://proxy.golang.org
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV GOARCH=amd64
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+RUN go build -a -o pluto *.go
```

Comment:
There are more flags necessary here to make it build like the normal binary. In particular, you'll want -s and -w, as well as passing the version and commit as build args.

https://github.com/FairwindsOps/pluto/blob/81c8d3298a4415075750d4ccdb1580b15c9b972e/Makefile#L12

Reply:
Another example in Goldilocks:

https://github.com/FairwindsOps/goldilocks/blob/master/Dockerfile

Reply:
Ahh nice, didn't see the make file. I'm just running make `build-linux` now. Do I need to pass in `VERSION` etc?

Also looks like there's a `build-docker` already, seems to be working

Reply:
I think you'll need to add the ARG directive to the dockerfile so that it gets picked up on build to make the build-docker work.

Reply:
Oh cool, I'd never seen `ARG` before. Did I do it right?